### ISSUE TITLE
Better support for multiple documents

### ISSUE THREAD

**Issue by paulrobertlloyd** [fbrctr/fabricator#123]
Currently, markdown docs are presented together on one page. Out of the box (or with minimal config), it would be nice to have docs viewable on a per page basis, i.e. better support for documentation that may consist of many (nested) pages. Currently, as far as I can see, achieving this requires creating a separate template for each page of documentation – please correct me if I’m wrong!

**Comment by LukeAskew** [comment]
Not a bad idea. I think the single page concept has just persisted since my initial concept.

Can you help flesh this out a little...

Given this setup:

```
└── docs
  ├── foo.md
  └── bar.md
```

Each doc would compile to `dist/docs/{filename}.html`

I think something like this might even be nice:

```
└── docs
  ├── javascript
    ├── foo.md
    └── index.md
  └── index.md
```

Where the compiled output looks like:

```
└── docs
  ├── javascript
    ├── foo.html
    └── index.html
  └── index.html
```


I almost think this should be the default, as past real-world Fabricator-generated documentation pages end up being fairly large. This change would make them easier to navigate.

**Comment by paulrobertlloyd** [comment]
Precisely. Support for this out of the box would be tremendously helpful. For example, my team already has a collection of markdown docs in our repo, and across nested folders. It would be nice to have these rendered by Fabricator without too much faffing about!
